### ISSUE TITLE
Error with containers

### ISSUE THREAD

**Issue by Piengineer12** [Piengineer12/public-addons#242]
On respawn:
```
lua/entities/isawc_container_base.lua:112: attempt to index local 'creator' (a nil value)
1. unknown - lua/entities/isawc_container_base.lua:112
2. Spawn - [C]:-1
3. fn - lua/autorun/isawc_base.lua:6372
4. unknown - lua/ulib/shared/hook.lua:109
```

**Comment by Piengineer12** [comment]
Seems to be related to [https://steamcommunity.com/sharedfiles/filedetails/?id=842747334](steam://openurl/https://steamcommunity.com/sharedfiles/filedetails/?id=842747334)

**Comment by Piengineer12** [comment]
Resolved via a55e106a3921e528cc318ee378cff70aa6725e3b.
